packages/discovery-react-components/src/components/DocumentPreview/components/PdfHighlight/PdfHighlight.tsx

```diff
@@ -1,15 +1,22 @@
-import React, { FC, useMemo, useEffect, useRef } from 'react';
+import React, { FC, useMemo, useEffect, useRef, useState, useCallback, MouseEvent } from 'react';
```

Comment:
I noticed that you have a bunch of code changes in PDF-related components, which sticks out as odd since this PR is specifically for the Text view. Are these carry-overs from changes in your PDF PR that haven't been merged with this one?

Reply:
Not applicable anymore. I corrected the alignment of this branch with master.